docker/development.yml

```diff
@@ -12,6 +12,23 @@ version: '3'
 services:
   # API Gateway
   nginx:
+    build:
+      context: ../src/web
+      cache_from:
+        - docker.cdot.systems/nginx:buildcache
+      # next.js needs build-time access to a number of API URL values, forward as ARGs
+      args:
+        # Web front-end URL
+        - WEB_URL=${WEB_URL}
+        # Telescope 1.0 API URL
+        - API_URL=${API_URL}
+        # Telescope 2.0 Microservice URLs
+        - IMAGE_URL=${IMAGE_URL}
+        - SSO_URL=${SSO_URL}
+        - POSTS_URL=${POSTS_URL}
+        - SEARCH_URL=${SEARCH_URL}
+        - FEED_DISCOVERY_URL=${FEED_DISCOVERY_URL}
+        - STATUS_URL=${STATUS_URL}
```

Comment:
This section is common for both development and production, I think it should be in `docker-compose.yml`, otherwise `production.yml` will not get this args.

Reply:
Except we don't build in production anymore, right?  They should already be embedded in the image.

Reply:
Oh! I forgot we have [this](https://github.com/Seneca-CDOT/telescope/blob/aecd7cb0e43c00c918504a8d42c05b52218d6c04/.github/workflows/docker-build-and-push.yml#L76) now. I really need to get used this new setting.